File: Des.web/Desenvolvimento web/python/algoritmo1.py
# escrever um algoritmo que recebe um numero informado pelo usuario. verifique e escreve se esse numero
# é positivo , negativo ou igual a 0
from cgitb import html

html

numero = (input("Informe um número: "))
if (numero > 0):
    print("Positivo")
elif (numero < 0):
    print("Negativo")
else:
    print("Igual a 0")

File: Des.web/Desenvolvimento web/php/algoritmo1.php
<?php
    
    $num = -5; intval($_GET['num']);
    if($num &lt; 0){
        echo "O numero é negativo";
    }elseif($num == 0){
        echo "O numero é igual a 0";
    }else{
        echo "O numero é positivo";
    }

?>
